dev/com.ibm.websphere.appserver.features/visibility/public/jcaInboundSecurity-1.0/com.ibm.websphere.appserver.jcaInboundSecurity-1.0.feature

```diff
@@ -4,10 +4,13 @@ visibility=public
 IBM-API-Package: javax.security.auth.message.callback; type="spec"
 IBM-ShortName: jcaInboundSecurity-1.0
 Subsystem-Name: Java Connector Architecture Security Inflow 1.0
--features=com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:=1.2, \
- com.ibm.websphere.appserver.security-1.0, \
- com.ibm.websphere.appserver.jca-1.6; ibm.tolerates:=1.7
--bundles=com.ibm.ws.jca.inbound.security, \
- com.ibm.websphere.javaee.jaspic.1.1; location:=dev/api/spec/; mavenCoordinates="javax.security.auth.message:javax.security.auth.message-api:1.1"
+-features=\
+   com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:=1.2, \
```

Comment:
Do you need to add:

com.ibm.websphere.appserver.builtinAuthentication-1.0 to this feature and  com.ibm.websphere.appserver.builtinAuthentication-2.0 to the 2.0 feature so that the tolerates in com.ibm.websphere.security-1.0 does the right thing?

Reply:
After talking with Jared and running a package minify, we decided we didn't need to do this. If there are issues with packaging in the build we will revisit.